<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsDialog</class>
 <widget class="QDialog" name="settingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>367</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>367</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>367</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ButtonApply">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>334</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>334</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>341</width>
     <height>324</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Interface</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>0</y>
       <width>323</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string>User interface</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QFontComboBox" name="fontBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="currentText">
       <string>Cascadia Code</string>
      </property>
      <property name="currentIndex">
       <number>24</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="sizeFBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="minimum">
       <number>3</number>
      </property>
      <property name="value">
       <number>14</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelFB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Шрифт:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSFB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Размер шрифта:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Compiler</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>0</y>
       <width>323</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string>Path to compiler</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Путь к g++:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="linePath">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonBrowse">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>60</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
